zebedee-cms/src/main/java/com/github/onsdigital/zebedee/keyring/SecureKeyring.java

```diff
@@ -0,0 +1,14 @@
+package com.github.onsdigital.zebedee.keyring;
+
+import com.github.onsdigital.zebedee.user.model.User;
+
+public interface SecureKeyring {
+
+    /**
+     * Populate the Keyring from an unlocked {@link User#keyring()}
+     *
+     * @param user the user to use to populate the keyring.
```

Comment:
Reading this back feels a bit too wordy.... maybe this would read better:
`@param user that populates the keyring` ?